### PR TITLE
Add Tunizi Dataset

### PR DESCRIPTION
**Pull request by abhishekkrthakur**

### CODE REVIEW

File: datasets/tunizi/tunizi.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+            datasets.SplitGenerator(
+                name=datasets.Split.TRAIN,
+                gen_kwargs={"datapath": path},
+            )
+        ]
+
+    def _generate_examples(self, datapath):
+        sentence_counter = 0
+        with open(datapath, encoding="utf-8") as f:
+            for row in f:
```
Comment (patrickvonplaten) [inline datasets/tunizi/tunizi.py:67]: (nit) you can also use `for sentence_counter, row in enumerate(f):`

**Review by patrickvonplaten — APPROVED** [review]
LGTM if tests pass

File: datasets/tunizi/tunizi.py (hunk 1 above)
Comment (abhishekkrthakur) [reply datasets/tunizi/tunizi.py]: done

File: datasets/tunizi/README.md [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+- expert-generated
+language_creators:
+- found
+languages:
+- aeb
+- ar
+licenses:
+- unknown
+multilinguality:
+- other-tunizi
```
Comment (lhoestq) [inline datasets/tunizi/README.md:12]: is it ok to have that @yjernite ? Maybe consider that as one language and different dialects ?
aeb is tunisian arabic and ar is arabic.

**Review by lhoestq — APPROVED** [review]
LGTM thank you :) 

feel free to merge after we've resolved the question about the `multilinguality` tag

File: datasets/tunizi/README.md (hunk 2 above)
Comment (yjernite) [reply datasets/tunizi/README.md:12]: This should be `multilingual` with both tags under `languages`

Comment (abhishekkrthakur) [reply datasets/tunizi/README.md]: hopefully fixed now.

File: datasets/tunizi/README.md [hunk 3]
```diff
@@ -0,0 +1,139 @@
+---
+annotations_creators:
+- expert-generated
+language_creators:
+- found
+languages:
+- aeb
```
Comment (lhoestq) [inline datasets/tunizi/README.md:7]: wasn't it both `ar` and `aeb` ?

**Pull request merged**